Portfolio/index.html

```diff
@@ -4,6 +4,7 @@
     <title>Portfolio</title>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
 
   <body style="margin: 0; padding: 0">

```

Comment:
The 'h1' tags should not be nested inside the 'pre' tag as it may affect the styling and structure of the content.